robot-db/src/main/java/repositories/BookSaver.java

```diff
@@ -13,36 +13,35 @@
 import java.util.stream.Collectors;
 
 /**
- * Class responsible for Inserting Parsed Book into Database
+ * {@link BookSaver} saves book into the database
  */
 @Repository
 @Slf4j
-public class BookInput {
+public class BookSaver {
 
     @Autowired
-    private AuthorInput authorInput;
+    private AuthorManager authorManager;
 
     @Autowired
-    private CategoryInput categoryInput;
+    private CategoryManager categoryManager;
 
     @Autowired
-    private PrintHouseInput printHouseInput;
+    private PrintHouseManager printHouseManager;
 
     @Autowired
     private BookDAO bookDAO;
 
     /**
-     * Saves the book in BookDAO object.
-     *
-     * @param parsedBook - book obtained from Parser.
+     * saves an {@link ParsedBook} into the database to achieve the goal
+     * it uses {@link AuthorManager}, {@link CategoryManager} and {@link PrintHouseManager}
+     * @param parsedBook - {@link ParsedBook} all Information about the book.
      */
     public void save (ParsedBook parsedBook) {
-        log.info("staring operations on database");
-        Category category = categoryInput.saveCategory(parsedBook.getCategory());
-        PrintHouse printHouse = printHouseInput.savePrintHouse(parsedBook.getPrintHouse());
+        Category category = categoryManager.getCategory(parsedBook.getCategory());
```

Comment:
It would be nice to make `getCategory()` from `CategoryManager` accepting `ParsedBook` object, ex.
before:
> Category category = categoryManager.getCategory(parsedBook.getCategory());
>PrintHouse printHouse = printHouseManager.getPrintHouse(parsedBook.getPrintHouse());

after:

` Category category = categoryManager.getCategory(parsedBook);`
`PrintHouse printHouse = printHouseManager.getPrintHouse(parsedBook);`

:ghost: